.github/workflows/main.yml

```diff
@@ -0,0 +1,59 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - '*'
+  pull_request:
+    branches:
+      - master
+      - '*'
+
+jobs:
+  molecule:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        scenario:
+          - amqp_connection
+          - amqp_default_interval
+          - collectd_connection
+          - default
+
+    env:
+      PY_COLORS: '1'
+      ANSIBLE_FORCE_COLOR: '1'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          path: "${{ github.repository }}"
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Pre-pull container
+        run: |
+          sudo docker pull docker.io/tripleomaster/centos-binary-collectd:current-tripleo-rdo
```

Comment:
Is there a quay.io equiv yet? This will eventually start to fail randomly likely due to new docker.io quotas.

Reply:
I'm not sure if that's there yet. I can go look it up when I respin this patch

Reply:
Cool, just noting it in case you start getting weird failures or whatever.